src/components/VersionSelect/styles.module.scss

```diff
@@ -10,3 +10,13 @@
     font-weight: 300;
   }
 }
+
+.simulatedFormControl {
+  align-items: center;
+  display: flex;
+
+  div {
```

Comment:
This is fragile because the internal structure of `Skeleton` could change. I'd like to see the code use a custom class like:

```tsx
<div className={makeClassName(styles.simulatedFormControl, ...)}>
  <Skeleton className={styles.simulatedFormControlSkeleton} />
</div>
```

With that, you can reference it directly:

```css
.simulatedFormControlSkeleton {
  height: auto;
  width: 60px;
}
```

If someone were working on the internal structure of `Skeleton`, they would see the `className` as a public prop which would make them think about all the components that might be depending on that class name.

Reply:
Oh nooes /o\

I changed laptop in the meantime and definitely did not want to push that change (and missed it when I self-reviewed this patch). This needs a CSS class of course.